app/views/declaration/additionalInformtion/additional_information_required.scala.html

```diff
@@ -14,42 +14,43 @@
  * limitations under the License.
  *@
 
-@import controllers.navigation.ItemId
-@import controllers.navigation.Navigator
 @import forms.common.YesNoAnswer.YesNoAnswers
 @import forms.common.YesNoAnswer
 @import models.requests.JourneyRequest
+@import controllers.navigation.Navigator
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @import views.{BackButton, Title}
 @import views.components.gds.Styles._
+@import controllers.navigation.ItemId
 @import views.html.components.gds._
 @import config.AppConfig
+@import forms.declaration.AdditionalInformationRequired
 
 @this(
-govukLayout: gdsMainTemplate,
-errorSummary: errorSummary,
-sectionHeader: sectionHeader,
-govukRadios: GovukRadios,
-exportsInputText: exportsInputText,
-govukDetails : GovukDetails,
-saveButtons: saveButtons,
-tariffDetails: tariffDetails,
-appConfig: AppConfig,
-formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+    govukLayout: gdsMainTemplate,
+    errorSummary: errorSummary,
+    sectionHeader: sectionHeader,
+    govukRadios: GovukRadios,
+    exportsInputText: exportsInputText,
+    govukDetails : GovukDetails,
+    saveButtons: saveButtons,
+    tariffDetails: tariffDetails,
+    appConfig: AppConfig,
+    formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 )
 
 @(mode: Mode, itemId: String, form: Form[YesNoAnswer])(implicit request: JourneyRequest[_], messages: Messages)
 
 @tradeTariffLink = {<a target="_blank" href=@appConfig.tradeTariffUrl>@messages("declaration.additionalInformationRequired.hint.link")</a>}
 
 @govukLayout(
-    title = Title("declaration.additionalInformationRequired.title"),
-    backButton = Some(BackButton(messages("site.back"), controllers.declaration.routes.CommodityMeasureController.displayPage(mode, (itemId))))) {
+    title = Title("declaration.additionalInformationRequired.title", "supplementary.items"),
+    backButton = Some(BackButton(messages("site.back"), Navigator.backLink(AdditionalInformationRequired, mode, ItemId(itemId))))) {
 
     @formHelper(action = controllers.declaration.routes.AdditionalInformationRequiredController.submitForm(mode, (itemId)), 'autoComplete -> "off") {
         @errorSummary(form.errors)
 
-        @sectionHeader(messages("supplementary.consignmentReferences.heading"))
+        @sectionHeader(messages("supplementary.items"))
```

Comment:
Just messages, can we use `declaration.items`?

Reply:
I've never bothered changing the section header message keys because I'm waiting for the day when we can change them all to the prototype "Secction X of Y" format.  Then we can name them properly.